### DISCUSSION TITLE
Parameters in the BSim Database Weight Template

### DISCUSSION THREAD

**Discussion by TongweiRen** [NationalSecurityAgency/ghidra#6668] [Q&A]
Hi there,

I noticed that there are weightnorm, probflip0, probflip1, probdiff0, probdiff1, addend, and scale, seven parameters are provided in the database weight template, may I ask how these values are calculated? I didn't find the code for tuning the weight template, did I miss anything?

Any help are appreciated! Thank you for your time!

**Comment by ghidracadabra** [comment]
A the moment the tuning code is not distributed.  It's not a bad idea; I'll bring this up with the team.

**Reply by landoty** [reply]
Hey there, 

I was curious if there was any progress on this? I've been working on a thesis project that would greatly benefit from this information! Thank you!
